src/main/java/com/esri/samples/map/create_and_save_map/README.md

```diff
@@ -20,19 +20,18 @@ chosen folder.</p>
   <li>Create an <code>ArcGISMap</code> with a <code>Basemap</code> and operational layers</li>
   <li>Create an <code>OAuthConfiguration</code> with your portal and app credential.</li>
   <li>Add the configuration to the <code>AuthenticationMangager</code></li>
-  <li>Create a <code>Portal</code> and load it. Use a custom <code>AuthenticationChallengeHandler</code> to 
-  authenticate with your username and password</li>
+  <li>Create a <code>Portal</code> and load it. Use a <code>DefaultAuthenticationChallengeHandler</code> to authenticate with your username and password</li>
```

Comment:
Whoops just spotted another original typo in the line above this: 
`AuthenticationMangager` -> `AuthenticationManager`